server/data/licenceApiClient.ts

```diff
@@ -130,6 +133,25 @@ export default class LicenceApiClient extends RestClient {
     )
   }
 
+  async addAdditionalCondition(
+    licenceId: string,
+    type: LicenceType,
+    additionalCondition: AddAdditionalConditionRequest,
+    user: User
+  ): Promise<AdditionalCondition> {
+    return (await this.post(
```

Comment:
just wondering if there was a practical reason for awaiting here but not on the delete?